great_expectations/rule_based_profiler/parameter_builder/numeric_metric_range_multi_batch_parameter_builder.py

```diff
@@ -531,10 +538,20 @@ def _get_bootstrap_estimate(
         else:
             n_resamples = num_bootstrap_samples
 
+        # Obtain bootstrap_random_seed override from "rule state" (i.e., variables and parameters); from instance variable otherwise.
+        random_seed: Optional[int] = get_parameter_value_and_validate_return_type(
+            domain=domain,
+            parameter_reference=kwargs.get("bootstrap_random_seed"),
+            expected_return_type=None,
+            variables=variables,
+            parameters=parameters,
+        )
+
         return compute_bootstrap_quantiles_point_estimate(
             metric_values=metric_values,
             false_positive_rate=false_positive_rate,
             n_resamples=n_resamples,
+            random_seed=random_seed,
```

Comment:
@NathanFarmer so basically here it would be called `bootstrap_random_seed` on the right hand side.  Thanks.

Reply:
@alexsherstinsky I would prefer to change the comment and leave it as `random_seed` if you don't mind.